docs/Contributing/architecture/software/software-updates.md

```diff
@@ -0,0 +1,114 @@
+# Software Updates Architecture
```

Comment:
This article refers to functionality that does not exist in Fleet. Need to delete this. We can get part of the way there with existing policy based workflows, but the UX of that will be covered in public-facing guides, so any articles here should either link to those guides or add (accurate) impl-level context.

Reply:
Emptying and leaving headers.